api-reference/docstubs/api/user-reminderview.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "reminderView"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# reminderView
+
+Namespace: microsoft.graph
+
+
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /me/reminderView
+GET /users/{usersId}/reminderView
+GET /me/managedDevices/{managedDeviceId}/users/{userId}/reminderView
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}.Required|
+
+## Request body
+In the request URL, provide the following query parameters with values.
```

Comment:
AFAIK, function parameters are part of the query URL, and not part of the request body.
Per current [guidance](https://msgo.azurewebsites.net/add/document/guidelines/parameters.html), please see the live site user-reminderview.md as an example for how function parameters are documented.

Reply:
Yeah - caught this earlier.